modules/sspBCBidAdapter.js

```diff
@@ -153,22 +156,37 @@ const mapBanner = slot => {
 }
 
 const mapImpression = slot => {
-  const { adUnitCode, bidId, params } = slot;
-  const { id, siteId } = params || {};
+  const { adUnitCode, bidId, params = {}, ortb2Imp = {} } = slot;
+  const { id, siteId } = params;
+  const { ext = {} } = ortb2Imp;
+
+  /*
+     check max size for this imp, and check/store number this size was called (for current view)
+     send this info as ext.pbsize
+  */
+  const slotSize = slot.sizes.length ? slot.sizes.reduce((prev, next) => prev[0] * prev[1] <= next[0] * next[1] ? next : prev).join('x') : '1x1';
+  adSizesCalled[slotSize] = adSizesCalled[slotSize] ? adSizesCalled[slotSize] += 1 : 1;
```

Comment:
This seems to be assigning the value twice. `+= 1` can be replaced with `+ 1`

Reply:
Right! Fixing it now.